proto/event.proto

```diff
@@ -58,6 +58,7 @@ message Metric {
     AggregatedHistogram aggregated_histogram = 9;
     AggregatedSummary aggregated_summary = 10;
   }
+  string namespace = 11;
```

Comment:
Do we need a documentation note that new Vector can't send metrics to older Vector via the "vector" protocols due to this new field?

Reply:
Adding a field with a never-before-used tag number is backward-compatible. The older implementation will just ignore tags it doesn't know about.